addons/website/static/src/js/website_controller_page_listing_layout.js

```diff
@@ -0,0 +1,72 @@
+/** @odoo-module **/
+
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.WebsiteControllerPageListingLayout = publicWidget.Widget.extend({
+    selector: ".o_website_listing_layout",
+    disabledInEditableMode: false,
+    events: {
+        "change .listing_layout_switcher input": "_onApplyLayoutChange",
+    },
```

Comment:
Does that mean that the user can modify the view mode (grid/list) while in edit mode?
It's weird

Reply:
I'm not sure I understand. By testing, that function is only triggered while NOT in edit mode. I think the code is copied from website_sale: _onApplyShopLayoutChange

Reply:
`disabledInEditableMode` means that the public widget will start also in edit mode (which is not the case by default).
It's explicitely asked to start in edit mode here